Automattic/newspack-newsletters
# fix: typos on "support" layout

### All Submissions:






### Changes proposed in this Pull Request:

Fix typos in "Support" layout:

```diff

<p>Dear *|FNAME|*,</p>



-<p>This is where you’d make a personal appeal to your readers. These types of emails are typically a few hundred words long, with a few grafs explaining the nature of the appeal and why readers should support your organization.</p>
+<p>This is where you’d make a personal appeal to your readers. These types of emails are typically a few hundred words long, with a few graphs explaining the nature of the appeal and why readers should support your organization.</p>



<p>You can send these emails as one-off emails to readers during a fundraising campaign, or you can build them into your onboarding flow to try to convert new readers into paying supporters.</p>



-<p>These emails can include grafs, or sometimes bullet points with key items listed out.</p>
+<p>These emails can include graphs, or sometimes bullet points with key items listed out.</p>



-<p>At the end, make sure you have a clear to call to action. Think carefully about the actual message. You could try something simple, like “Support us” or “Donate now,” or something more specific, like, “Yes, I want to support local journalism!” Try several different CTAs and see what works best.</p>
+<p>At the end, make sure you have a clear call to action. Think carefully about the actual message. You could try something simple, like “Support us” or “Donate now,” or something more specific, like, “Yes, I want to support local journalism!” Try several different CTAs and see what works best.</p>

``` 

### How to test the changes in this Pull Request:

1. Switch to this branch, run `npm run build`
2. Create a newsletter using the "Support" layout
3. Observe the changes diffed above

### Other information:
